arch/arm/boot/dts/overlays/revpi-core-se-dt-blob-overlay.dts

```diff
@@ -0,0 +1,1791 @@
+            pin_define@BT_ON {
+               type = "internal";
+               number = <45>;
+            };
+            pin_define@WL_ON {
+               type = "internal";
+               number = <41>;
+            };
+            pin_define@WL_LPO_CLK {
+               type = "internal";
+               number = <43>;
+            };
+            pin_define@SMPS_SDA {
+               type = "absent";
+            };
+            pin_define@SMPS_SCL {
+               type = "absent";
+            };
+            pin_define@ETH_CLK {
+               type = "absent";
+            };
+            pin_define@USB_LIMIT_1A2 {
+               type = "absent";
+            };
+            pin_define@SIO_1V8_SEL {
+               type = "absent";
+            };
+            pin_define@PWML {
+               type = "absent";
+            };
+            pin_define@PWMR {
+               type = "absent";
+            };
+            pin_define@SAFE_MODE {
+               type = "absent";
+            };
+            pin_define@SD_CARD_DETECT {
+               type = "absent";
+            };
+            pin_define@ID_SDA {
+               type = "internal";
+               number = <0>;
+            };
+            pin_define@ID_SCL {
+               type = "internal";
+               number = <1>;
+            };
+            pin_define@DISPLAY_I2C_PORT {
+               type = "absent";
+            };
+            pin_define@DISPLAY_SDA {
+               type = "absent";
+            };
+            pin_define@DISPLAY_SCL {
+               type = "absent";
+            };
+         }; // pin_defines
+      }; // pins
+
+      pins_cm {
+         pin_config {
+            pin@default {
+               polarity = "active_high";
+               termination = "pull_down";
+               startup_state = "inactive";
+               function = "input";
+            }; // pin
+            pin@p14 { function = "uart0";  termination = "no_pulling"; }; // TX uart0
+            pin@p15 { function = "uart0";  termination = "pull_up"; }; // RX uart0
+            pin@p47 { function = "output"; termination = "no_pulling"; polarity = "active_low"; }; // EMMC_ENABLE_N
+            pin@p48 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD CLK
+            pin@p49 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD CMD
+            pin@p50 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD D0
+            pin@p51 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD D1
+            pin@p52 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD D2
+            pin@p53 { function = "sdcard"; termination = "pull_up";    drive_strength_mA = < 8 >; }; // SD D3
+            pin@p39 { function = "spi"; termination = "no_pulling";    drive_strength_mA = < 8 >; }; // RevPi Core SPI0 CLK
```

Comment:
This is only used for CM1 modules. The SE variants will only use CM4s'.
```suggestion
```

Reply:
the node pins_cm, it was just copied from Core dt blob overlay. And based on Core dt blob overlay, the pin@39 in pins_cm3 and pins_cm4s have be removed. 
Btw, form the following links, it seems the SE variants will also use CM3:
https://kunbus-gmbh.atlassian.net/wiki/spaces/EN/pages/2550956171/2022-03-22+RevPi+Pi+Core+SE#Spezifikationen

Reply:
Hmmm, we should drop the `revpi-core-se-dt-blob-overlay.dts` completely. It will not be used as the Core SE gets a HAT EEPROM. We can use the dt-blob from the foundation here.

Also the SE variants will only be shipped with CM4s modules. Thus the S in the name.

Reply:
So both revpi-core-se-dt-blob-overlay.dts and revpi-connect-se-dt-blob-overlay.dts are not needed and should be dropped?

Reply:
No only the `revpi-core-se-dt-blob-overlay.dts` should be dropped. The Connect SE will be based on the regular Connect board and doesn't have a hat eeprom.

Reply:
As I understood from following text of HAT, the DT blob is still needed, just not read it from /boo/dt-blob.bin, but from HAT EEPROM
https://github.com/raspberrypi/hats/blob/master/devicetree-guide.md#pinctrl-nodes-vs-gpio-map
If I were right, we should still have a "revpi-core-se-dt-blob-overlay.dts", and compile it. However, instead of installing it along with kernel package, we should flash it. pls correct me if I am wrong.

Reply:
This is not how it works. The HAT EEPROM has a special format to set the first GPIO bank. https://github.com/raspberrypi/hats/blob/master/eeprom-format.md#gpio-map-atom-data-type0x0002

Reply:
Please drop the dt-blob for the RevPi Core SE, we will also not install the dtbob for the RevPi Core redesign.